.ci/validation.cmd

```diff
@@ -9,6 +9,12 @@
 SET OPTION=%1
 
 if "%OPTION%" == "sevntu" (
+  call mvn -e --no-transfer-progress verify -Pno-validations,sevntu^
+    || goto :ERROR
+  goto :END_CASE
+)
+
+if "%OPTION%" == "verify_with_checkstyle" (
   call mvn -e --no-transfer-progress verify -DskipTests -DskipITs -Dpmd.skip=true^
```

Comment:
Why we need this job now?

Reply:
This and the changes to appveyor.yml is to keep CI running the same things as before my change.

Before when we ran `verify` phase, it ran our ANT runs and maven-checkstyle-plugin run all in 1 go.

Now because we had to move sevntu run to a profile, it requires 2 separate commands.

I thought this would be better than combining both calls and results into 1 appveyor run. Similar to how we have other CIs split simple things out into different runs.

Reply:
Name is very confusing. I am not sure how to name it better

Reply:
If we need only checkstyle to be executed, can we do 
https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-checkstyle-report

Reply:
better to rename as suggested above.
and update command to make only ANT execution of checksyle.

Reply:
> name it run_checkstyle
> only checkstyle to be executed, can we do

Both are done.